core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -602,6 +606,36 @@ class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with Matchers
     }
   }
 
+  test("acls with downloading files") {
```

Comment:
Should this test just be a new URL added to `testUrls` in "ui and api authorization checks"? What is it doing that is different than what that test does and why?